onboarder/patch_and_minor/update_ee_on_premise.rst

```diff
@@ -66,7 +65,9 @@ You can now clear the Symfony cache, re-install assets and rebuild the front-end
     service php7.2-fpm start
```

Comment:
I think we can change line 64 to `rm -rf var/cache/* ./public/bundles/* ./public/css/* ./public/js/*` too

Reply:
Indeed, fix! thanks